backend/app/services/file_service.py

```
@@ -513,4 +514,178 @@ async def process_uploaded_file(file: UploadFile) -> str:
         except Exception as e:
             # 异常情况下也使用安全的文件清理方法
             FileService._safe_file_cleanup(file_path)
-            raise e
\ No newline at end of file
+            raise e
+
+    # ============ 素材解析：文本与图片分离 ============
+
+    @staticmethod
+    def _save_material_image(image_data: bytes, ext: str, material_id: str, index: int) -> str:
+        """将单张素材图片保存到磁盘，返回相对路径"""
```

Ruff (0.15.9)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)